public/components/stub-modal/stub-modal.html

```diff
@@ -41,7 +41,7 @@ <h4 class="modal-title">{{ modalTitle }}</h4>
                     </select>
                 </div>
             </div>
-            <div class="form-group pull-left col-xs-8" ng-if="mode !== 'import' && contentName !== 'Atom'">
+            <div class="form-group pull-left col-xs-8" ng-if="contentName !== 'Atom'">
```

Comment:
I'm not sure if this is hiding the Commissioning part of the form when importing atoms.